File: src/main/java/cz/agents/amodsim/config/Rebalancing.java
package cz.agents.amodsim.config;

import java.lang.Double;
import java.lang.Integer;
import java.lang.String;

public final class Rebalancing {
  public final Integer maxWaitInQueue = 0;

  public final String loadShapes = "true";

  public final String filePath = "/home/fido/AIC data/Shared/amod-data/agentpolis-experiment/prague/stations_40/stations";

  public final Double vehCoef = 1.0;

  public final String useSmoothedDemand = "true";

  public final String policyFilePath = "/home/fido/AIC data/Shared/amod-data/agentpolis-experiment/prague/stations_40/policy.json";

  public final String method = "emd-s";

  public final Integer timestep = 600;

  public final Integer vehicleLimit = 0;

  public final String type = "stations";

  public Rebalancing() {
  }
}

File: src/main/python/amodsim/config/rebalancing.py

class Rebalancing:
    def __init__(self, properties: dict=None):
        self.timestep = properties.get("timestep")
        self.type = properties.get("type")
        self.file_path = properties.get("file_path")
        self.use_smoothed_demand = properties.get("use_smoothed_demand")
        self.max_wait_in_queue = properties.get("max_wait_in_queue")
        self.method = properties.get("method")
        self.vehicle_limit = properties.get("vehicle_limit")
        self.veh_coef = properties.get("veh_coef")
        self.load_shapes = properties.get("load_shapes")
        self.policy_file_path = properties.get("policy_file_path")


        pass

